### ISSUE TITLE
update tcc regularly

### ISSUE THREAD

**Issue by SleepyRoy** [vlang/v#12001]
Unlike other more mature compilers such as gcc or clang, tcc is at stage beta (https://savannah.nongnu.org/projects/tinycc) and thus receives function updates more frequently. The newer tcc has greater probability to work nicely with V. Therefore, it would be nice (to have a mechanism) to update tcc regularly.  
As an example, I just tried `v -cc tcc -gc boehm` on windows with tcc and it told me `undeclared __ATOMIC_SEQ_CST` etc. However, by watching history of commits of tcc there was some work on this recently (like on 20210128 https://repo.or.cz/tinycc.git/commit/458457590db88c7a29f2349a499434f4e756367e) and it might actually fix the bugs.

**Comment by Cons-Cat** [comment]
Can't you just get TCC from a package manager and update it using that?

**Comment by JalonSolov** [comment]
No, because we use the latest commit, not what's been packaged.  The package are often years out of date.
